config/default_configs/default_config.yml

```
@@ -220,6 +220,9 @@ orographic_gravity_wave:
 non_orographic_gravity_wave:
   help: "Apply parameterization for convective gravity wave forcing on horizontal mean flow [`false` (default), `true`]"
   value: false
+dt_nogw:
+  help: "Time between calling non-orographic gravity wave update"
```

```suggestion
  help: "Time between calling non-orographic gravity wave update. Ignored if `non_orographic_gravity_wave` is not set to `true`."
```

If there's anything else you want to add here, please go ahead. Best to have more information than less (don't emulate the other `help` sections in this file).

---

Thanks for pointing this out. As mentioned during the Monday meeting, I will go through the documentations thoroughly after I am done with the orographic GW module. This is because things might change. E.g., we may have just one dt_gw in the end, for example.